src/com/android/calendar/Utils.java

```diff
@@ -196,6 +196,13 @@ public static boolean isJellybeanOrLater() {
       return Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN;
     }
 
+    /**
+     * Returns whether the SDK is the KeyLimePie release or later.
+     */
+    public static boolean isKeyLimePieOrLater() {
+      return Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN_MR2;
```

Comment:
As you see, we have API 19, so I expect more stuff to brake if someone would try. So +1 for removing it.